server.js

```diff
@@ -49,15 +49,27 @@ app.post('/api/register', async (req, res) => {
   }
 });
 
-app.listen(port, () => {
-  console.log(`Server is running on port ${port}`);
-});
-
 // app.post('/api/achievement', async (req, res) => {
 //   console.log("It's alive!");
 //   res.status(200).json({ message: 'Achievement successful' });
 // });
 
+app.post('/api/attempt', async (req, res) => {
+  try {
+    const { characters_attempted, characters_missed, wpm, email } = req.body;
+    // console.log({ characters_attempted, characters_missed, wpm, email })
+    const getUserId = await getUserIdByEmail(email);
+    const result = insertAttempt(getUserId, characters_attempted, characters_missed, wpm);
```

Comment:
Nit: I don't like the name `insertAttempt`... If you just named it `insert` it is obvious that you are _attempting_ to insert something. Failure is always a possibility. 

I prefer `insert`, or `insertRecord`.
```suggestion
    const result = insert(getUserId, characters_attempted, characters_missed, wpm);
```

Reply:
I agree that insertAttempt is not a super clear name, what do you think about insertGameAttemptData? We have multiple functions that perform an insert query, perhaps this would be more clear. Thanks for the input!

Reply:
I am open to that. What about some thing a bit simpler? What about 'insertGame'? And then you could name the table, the "game" table.

Something like that feels simple and intuitive.